js/scripts.js

```diff
@@ -285,12 +286,20 @@ ontouchmove = onDragStart;
 onmouseup = onDragEnd;
 ontouchend = onDragEnd;
 
-function onDragReset() {
+function onDragReset(e) {
     dragged = false;
+    if (e.touches) {
+        touchStartPositon = [ e.touches[0].screenX, e.touches[0].screenY ];
+    }
 }
 
-function onDragStart() {
-    dragged = true;
+function onDragStart(e) {
+    if (!touchStartPositon || !e.touches || e.touches.length < 1) {
+        dragged = true;
+    } else if (Math.sqrt(Math.pow(touchStartPositon[0] - e.touches[0].screenX, 2)
```

Comment:
Merge the `if` and `else if`, separating with a || instead, because you're doing the same thing in both cases.